go-example/metrics/gooteltest/cmd/oteltester/main.go

```diff
@@ -16,12 +16,18 @@ import (
 	"go.opentelemetry.io/otel/metric/global"
 	"go.opentelemetry.io/otel/sdk/metric/aggregator/histogram"
 	controller "go.opentelemetry.io/otel/sdk/metric/controller/basic"
+	"go.opentelemetry.io/otel/sdk/metric/export/aggregation"
 	processor "go.opentelemetry.io/otel/sdk/metric/processor/basic"
 	"go.opentelemetry.io/otel/sdk/metric/selector/simple"
 	"go.opentelemetry.io/otel/sdk/resource"
 	semconv "go.opentelemetry.io/otel/semconv/v1.7.0"
-	"google.golang.org/grpc"
-	"google.golang.org/grpc/credentials/insecure"
+)
+
+const (
+	MetricTypeGauge          = "gauge"
+	MetricTypeSum            = "sum"
```

Comment:
Nit: I notice that you have similar constants in config.go.  Since you are already importing gooteltest, you could use the constants from config.go here instead of redefining them.  For example

```gooteltest.DeltaAggregationSelector```

What you have works, but defining the constants in one place helps with maintenance.